.gitlab-ci.yml

```diff
@@ -171,3 +173,31 @@ force_release_prod:
     RELEASE_COMMAND: scripts/release.sh staging prod --force
   only:
     - staging
+
+verify_sync_partial:on-schedule:
+  stage: verify_sync
+  only:
+    refs:
+      - schedules
+  script:
+    - FILES_SECRETS_NAME="${DSS_SECRETS_STORE}/${DSS_DEPLOYMENT_STAGE}/{DSS_FILES_VERIFY_SECRETS_NAME}"
+    - BLOBS_SECRETS_NAME="${DSS_SECRETS_STORE}/${DSS_DEPLOYMENT_STAGE}/{DSS_BLOBS_VERIFY_SECRETS_NAME}"
+    - BUNDLES_SECRETS_NAME="${DSS_SECRETS_STORE}/${DSS_DEPLOYMENT_STAGE}/{DSS_BUNDLES_VERIFY_SECRETS_NAME}"
+    - FILES_LAST_VERIFIED="$(python scripts/dss-ops.py secrets get --secret-names ${FILES_SECRETS_NAME} jq -r '.[]')"
+    - BLOBS_LAST_VERIFIED="$(python scripts/dss-ops.py secrets get --secret-names ${BLOBS_SECRETS_NAME} | jq -r '.[]')"
+    - BUNDLE_LAST_VERIFIED="$(python scripts/dss-ops.py secrets get --secret-names ${BUNDLES_SECRETS_NAME} | jq -r '.[]')"
+    - bash scripts/verify_consistency.sh files "${FILES_LAST_VERIFIED}" | tee file_verify_log
+    - bash scripts/verify_consistency.sh blobs "${BLOBS_LAST_VERIFIED}" | tee blob_verify_log
+    - bash scripts/verify_consistency.sh bundles "${BUNDLE_LAST_VERIFIED}" | tee bundle_verify_log
```

Comment:
possible to print out the logs here? i assume the get lost when the image is destroyed?

Reply:
Yes, `tee` will write the logs to both file and stdout